README.md

```diff
@@ -139,6 +139,13 @@ Add a button to import Takealot releases to MusicBrainz
 [![Source](https://github.com/jerone/UserScripts/blob/master/_resources/Source-button.png)](https://github.com/murdos/musicbrainz-userscripts/blob/master/takealot_importer.user.js)
 [![Install](https://raw.github.com/jerone/UserScripts/master/_resources/Install-button.png)](https://raw.github.com/murdos/musicbrainz-userscripts/master/takealot_importer.user.js)
 
+## <a name="facebook_importer"></a> Import Facebook Events to MusicBrainz
```

Comment:
The `name` should be `facebook_events_importer` (to be consistent with the rest).